.github/workflows/push.yml

```diff
@@ -0,0 +1,23 @@
+on: push
+
+name: CI / GitHub Actions
```

Comment:
Let's maybe just say
```
name: Continuous Integration
```
for clarity

Reply:
b36f14514b76c6d6480c64cc71b55e1c4110de10 🤫